src/pages/dmt5-trading-signals/_subscription.js

```diff
@@ -0,0 +1,288 @@
+import React from 'react'
+import styled from 'styled-components'
+import { graphql, useStaticQuery } from 'gatsby'
+import { Text, Header, QueryImage, LocalizedLinkText } from 'components/elements'
+import { localize, Localize } from 'components/localization'
+import { Container, SectionContainer, Flex } from 'components/containers'
+import { LinkButton } from 'components/form'
+import device from 'themes/device'
+
+const query = graphql`
+    query {
+        signals: file(relativePath: { eq: "dmt5_signals.png" }) {
+            ...fadeIn
+        }
+        statistics: file(relativePath: { eq: "dmt5_statistics.png" }) {
+            ...fadeIn
+        }
+        renew: file(relativePath: { eq: "dmt5_renew.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const StyledSectionContainer = styled(SectionContainer)`
+    margin: 0;
+    padding: 0;
+    background-color: var(--color-white);
```

Comment:
fyi, section container has props for background color, and padding, you can pass white in `background` attribute as well. but its fine too to make it in styled component